node_lambnik/src/tiler/src/util/build-xml.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * This script uses carto to compile templated MML files
+ * to XML that mapnik can understand
+ */
+
+import carto from 'carto'
+import path from 'path'
+
+import { readFile, writeFile } from './fs-promise'
+
+const IN_FILE = process.argv[2]
+const OUT_FILE = process.argv[3]
+const NODE_ENV = process.env.NODE_ENV === 'production'
+console.log(`Production mode=${NODE_ENV}`)
+
+// Takes in a string and templates in the proper env variables
+const fillTemplate = (mmlString) => {
```

Comment:
This defines its own basic and very specific templating syntax--converting bash-style `${VAR_NAME}` to values loaded from the environment matching the `VAR_NAME` but prefixed with the environment.

I think this makes sense--this is simple and understandable, and using an existing template library or trying to make it work as a simple template string would require more work to get what we want.

I mainly want to note that if our needs get more complicated, the point at which we should switch to a defined template system rather than add nuances to this implementation would come soon.

Also, I think it's worth adding a bit of detail to the comment on this function along the lines of this note.